nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `EnableDeadLetteringOnMessageExpiration(bool)`: Dead letters messages when they expire.
+ * `ForwardDeadLetteredMessagesTo(string)`: Forward all dead lettered message to the specified entity.
+ * `ForwardDeadLetteredMessagesTo(Func<string, bool>, string)`: Forward all dead lettered message to the specified entity if the given condition is true.
+ * `LockDuration(TimeSpan)`: The period of time that Azure Service Bus will lock a message before trying to redeliver it, defaults to 30 seconds.
+ * `MaxDeliveryCount(int)`: Sets the maximum delivery count, defaults to 10.
+ * `AutoDeleteOnIdle(TimeSpan)`: Auto deletes the entity if it hasn't been used for the given time frame.
+ * `DescriptionFactory(Func<string, string, ReadOnlySettings, SubscriptionDescription>)`: A factory method that allows to create a SubscriptionDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Subscription API.
+ 
+## Controlling Connectivity
+
+The following settings determine how NServiceBus will connect to Azure Service Bus.
+ 
+ * `NumberOfClientsPerEntity(int)`: NServiceBus maintains a small pool of receive and send clients for each entity, this setting determines how big that pool is. Defaults to 5.
+ * `ConnectivityMode(ConnectivityMode)`: Determines how NServiceBus connects to Azure Service Bus, using TCP or HTTP. Defaults to TCP. 
+ * `BrokeredMessageBodyType(SupportedBrokeredMessageBodyTypes)`: Controls how the body of a brokered will be serialized as a byte array or as a stream. Defaults to byte array.
+ * `MessagingFactories()`: Provides access to settings of MessagingFactory instances used under the hood. These settings will automatically apply to all MessageReceiver and MessageSender instances created by the MessagingFactory.
+ * `MessageReceivers()`: Provides access to the settings of the MessageReceiver instances.
+ * `MessageSenders()`: Provides access to the settings of the MessageSender instances.
+
+### Messaging Factories
+
+Messaging factories are the heart of connectivity management in the Azure Service Bus SDK, each messaging factory maintains a TCP connection with the broker and creates MessageSender and MessageReceiver instances for that connection. This implies that all senders and receivers created by the same factory, use the same underlying TCP connection and inherit the settings configured at the messaging factory level. The following settings allow to control the messaging factories.
+
+ * `NumberOfMessagingFactoriesPerNamespace(int)`: NServiceBus maintains a small pool of messaging factories per namespace, this setting determines the size of the pool. Defaults to 5.
+ * `RetryPolicy(RetryPolicy)`: Determines how entities should respond on transient connectivity failures. Defaults to `RetryPolicy.Default`, which is an exponential retry under the hood.
```

Comment:
> an exponential retry

you mean retry at exponentially longer time intervals?

I've found expression "exponential backoff", so seems similar enough, if you prefer keep that one.